### DISCUSSION TITLE
tz_info support for migrations

### DISCUSSION THREAD

**Discussion by thorin-schiffer** [BeanieODM/beanie#905] [Feature Request]
Migrations use it's own client, so no additional options can be passed to the client (like tz_info).
Solution: add an option to migrate command, enabling the tz_info option on the motor client used in beanie.migrations.database.DBHandler.set_db

**Comment by AndreyNvk** [comment]
That's also an issue for me. When I use `pydantic.AwareDatetime` in models for migrations, the migration fail, because there is no info on timezone, even though it present in the database.

Please, consider adding motor client as a parameter of the migrations, so we can have the full control over its configuration (like tz_info)

**Comment by pschoen-itsc** [comment]
I support this too. I also think that it should be possible to completely customize the motor client. There are a lot of options that can be set and providing a way to use all of them would be nice.
